src/main/java/com/evanlennick/retry4j/CallExecutor.java

```diff
@@ -153,6 +154,32 @@ private void postExecutionCleanup(Callable<T> callable, int maxTries, AttemptSta
         return attemptStatus;
     }
 
+    private boolean isOneOfValuesToExpect( T callResult ) {
+        Object[] valuesToExpect = config.getValuesToExpect();
```

Comment:
could you format these blocks to be consistent with code styling in the rest of the class? eg: less whitespace and line breaks

Reply:
Updated.